R-package/man/FastRGF_Classifier.Rd

```diff
@@ -17,48 +16,9 @@
-
-\item{n_estimators}{an integer. The number of trees in the forest (Original name: forest.ntrees.)}
-
-\item{max_depth}{an integer. Maximum tree depth (Original name: dtree.max_level.)}
-
-\item{max_leaf}{an integer. Maximum number of leaf nodes in best-first search (Original name: dtree.max_nodes.)}
-
-\item{tree_gain_ratio}{a float. New tree is created when leaf-nodes gain < this value * estimated gain of creating new tree (Original name: dtree.new_tree_gain_ratio.)}
-
-\item{min_samples_leaf}{an integer or float. Minimum number of training data points in each leaf node. If an integer, then consider min_samples_leaf as the minimum number. If a float, then min_samples_leaf is a percentage and ceil(min_samples_leaf * n_samples) are the minimum number of samples for each node (Original name: dtree.min_sample.)}
-
-\item{loss}{a character string. One of \emph{"LS"} (Least squares loss), \emph{"MODLS"} (Modified least squares loss) or \emph{"LOGISTIC"} (Logistic loss) (Original name: dtree.loss.)}
-
-\item{l1}{a float. Used to control the degree of L1 regularization (Original name: dtree.lamL1.)}
-
-\item{l2}{a float. Used to control the degree of L2 regularization (Original name: dtree.lamL2.)}
-
-\item{opt_algorithm}{a character string. Either \emph{"rgf"} or \emph{"epsilon-greedy"}. Optimization method for training forest (Original name: forest.opt.)}
-
-\item{learning_rate}{a float. Step size of epsilon-greedy boosting. Meant for being used with opt_algorithm = "epsilon-greedy" (Original name: forest.stepsize.)}
-
-\item{max_bin}{an integer or NULL. Maximum number of discretized values (bins). If NULL, 65000 is used for dense data and 200 for sparse data (Original name: discretize.(sparse/dense).max_buckets.)}
-
-\item{min_child_weight}{a float. Minimum sum of data weights for each discretized value (bin) (Original name: discretize.(sparse/dense).min_bucket_weights.)}
-
-\item{data_l2}{a float. Used to control the degree of L2 regularization for discretization (Original name: discretize.(sparse/dense).lamL2.)}
-
-\item{sparse_max_features}{an integer. Maximum number of selected features. Meant for being used with sparse data (Original name: discretize.sparse.max_features.)}
-
-\item{sparse_min_occurences}{an integer. Minimum number of occurrences for a feature to be selected. Meant for being used with sparse data (Original name: discretize.sparse.min_occrrences.)}
-
-\item{calc_prob}{a character string. Either \emph{"sigmoid"} or \emph{"softmax"}. Method of probability calculation}
-
-\item{n_jobs}{an integer. The number of jobs to run in parallel for both fit and predict. If -1, all CPUs are used. If -2, all CPUs but one are used. If < -1, (n_cpus + 1 + n_jobs) are used (Original name: set.nthreads.)}
-
-\item{verbose}{an integer. Controls the verbosity of the tree building process (Original name: set.verbose.)}
-}
 \description{
+A Fast Regularized Greedy Forest classifier
+
 A Fast Regularized Greedy Forest classifier
```

Comment:
Are these duplicated lines expected or it is a bug?

Reply:
I'm not sure. The previous .Rd file consisted of a [single line](https://github.com/RGF-team/rgf/blob/master/R-package/man/FastRGF_Classifier.Rd#L61-L63). The .Rd file is the manual page of the corresponding .R file. It has to do with the [roxygen2](https://github.com/RGF-team/rgf/blob/master/R-package/DESCRIPTION#L24) package and how it builds the manual pages from the .R files (I've used the *roxygen2* 7.0.2 version for this PR)